cmudb/tscout/clang_parser.py

```diff
@@ -49,13 +81,42 @@ class ClangParser:
     """
 
     def __init__(self):
-        # Parse the translation unit.
-        index = clang.cindex.Index.create()
-        translation_unit = index.parse(CLANG_EXECNODES_H, args=CLANG_ARGS)
+        indexes: List[clang.cindex.Index] = []
```

Comment:
@mbutrovich I added types, some comments, and reformatted a bit to make `pycodestyle` happy (80 char).
I don't have a strong opinion on 80char vs 100char vs 120char as long as we're consistent, sticking with Python convention for now.